toolbox

```diff
@@ -832,12 +831,21 @@ init_container()
             fi
         fi
 
+        groups=
```

Comment:
Sadly, the idea behind PR #165 wasn't very sound, so we have to do the same thing for `podman create` too.

Reply:
Done.